LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# Licenses
+
+## Specification License
+
+Specifications in the repository are subject to the **Community Specification License 1.0** available at [https://github.com/CommunitySpecification/1.0](https://github.com/CommunitySpecification/1.0).
```

Comment:
I think we need to link to the

[FINOS Community Specification](https://github.com/finos/standards-project-blueprint/)

which is indeed the copy, with minor / non breaking changes (to the governance, not to the license) that the board approved as technical charter (@scooter99boston).

Our goal is to keep it in line with the upstream CSL, so we can keep calling it CSL, but for clarity I think it's important to link the appropriate FINOS downstream copy (which has for example the Apache source code as default provision in it).